public/assets/heistscene.svg

```diff
@@ -0,0 +1,220 @@
+<feBlend mode="normal" in2="effect1_innerShadow_1470_1427" result="effect2_innerShadow_1470_1427"/>
+</filter>
+<filter id="filter2_f_1470_1427" x="226.161" y="366.91" width="429.916" height="262.033" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape"/>
+<feGaussianBlur stdDeviation="31.4869" result="effect1_foregroundBlur_1470_1427"/>
+</filter>
+<filter id="filter3_f_1470_1427" x="326.573" y="315.974" width="290.845" height="284.272" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape"/>
+<feGaussianBlur stdDeviation="3.63072" result="effect1_foregroundBlur_1470_1427"/>
+</filter>
+<filter id="filter4_dddd_1470_1427" x="-26.2066" y="-65.8785" width="981.71" height="981.71" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+<feOffset/>
+<feGaussianBlur stdDeviation="11.0556"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.490196 0 0 0 0 0.682353 0 0 0 0 0.831373 0 0 0 1 0"/>
+<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow_1470_1427"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+<feOffset/>
+<feGaussianBlur stdDeviation="22.1113"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.490196 0 0 0 0 0.682353 0 0 0 0 0.831373 0 0 0 1 0"/>
+<feBlend mode="normal" in2="effect1_dropShadow_1470_1427" result="effect2_dropShadow_1470_1427"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+<feOffset/>
+<feGaussianBlur stdDeviation="77.3895"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.490196 0 0 0 0 0.682353 0 0 0 0 0.831373 0 0 0 1 0"/>
+<feBlend mode="normal" in2="effect2_dropShadow_1470_1427" result="effect3_dropShadow_1470_1427"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+<feOffset/>
+<feGaussianBlur stdDeviation="154.779"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.490196 0 0 0 0 0.682353 0 0 0 0 0.831373 0 0 0 1 0"/>
+<feBlend mode="normal" in2="effect3_dropShadow_1470_1427" result="effect4_dropShadow_1470_1427"/>
+<feBlend mode="normal" in="SourceGraphic" in2="effect4_dropShadow_1470_1427" result="shape"/>
+</filter>
+<linearGradient id="paint0_linear_1470_1427" x1="428.438" y1="189.782" x2="428.438" y2="397.832" gradientUnits="userSpaceOnUse">
+<stop stop-color="white"/>
+<stop offset="1" stop-color="#999999" stop-opacity="0"/>
```

Comment:
there's a faint dark line here and I think it's because once it reaches opacity 0, the browser cuts off the rest of the shape, and so the black shadow gets applied and it shows up as a black line cutting through here

![image](https://github.com/user-attachments/assets/ae64e6c6-3824-4e71-9802-0c6bd5dfaad2)

it seems changing the stop opacity to something almost but not completely transparent fixes it

```suggestion
<stop offset="1" stop-color="#999999" stop-opacity="0.01"/>
```

Alternatively, you could just remove the drop shadow on "Vector 1" in Figma, which doesn't seem to change anything visually, and re-export the SVG. I think this option is probably better

![image](https://github.com/user-attachments/assets/dd7cb83d-0b4c-4010-8625-b2b159a712c7)